mdata/cache/accnt/flat_accnt.go

```diff
@@ -8,7 +8,27 @@ import (
 	"github.com/raintank/schema"
 )
 
-const evictQSize = 1000
+const (
+	evictQSize = 1000
+
+	// these numbers are estimates and could still use some fine tuning
+	// they do not account for map growth
+
+	// EvictTarget (24 bytes + 4 bytes) (AMKey, uint32) + 8 bytes (pointer from map[interface{}]*EvictTarget)
+	// + 40 bytes (Element in List)
+	lruItemSize = 76
+
+	// k: 4 bytes + v: 8 bytes (map[uint32]uint64)
+	flatChunkSize = 12
```

Comment:
It would be `flatAccntMetChunkSize` which is long. All of those names are actually pretty long, open to suggestions for abbreviations.
What about something like `famChunkSize`, `famSize`, `ccmSize`, and `ccmChunkSize`?
I think `lruItemSize` is already short enough.

Reply:
yeah, you're suggestions look good, and they're consistent